pkg/controllers/v1/mpi_job_controller.go

```diff
@@ -88,6 +88,14 @@ const (
 	// fails to sync due to dependent resources already existing.
 	MessageResourceExists = "Resource %q of Kind %q already exists and is not managed by MPIJob"
 
+	// ErrResourceDoesNotExists is used as part of the Event 'reason' when some
+	// resource is missing in yaml
+	ErrResourceDoesNotExists = "ErrResourceNotExists"
```

Comment:
Should be "exist", not "exists". Also the string value should be changed.

Reply:
done.:sweat_smile: